tests/cancel-test.ts

```diff
@@ -146,180 +146,6 @@ QUnit.test('cancel during flush', function(assert) {
   assert.ok(!functionWasCalled, 'function was not called');
 });
 
-QUnit.test('with GUID_KEY and target', function(assert) {
```

Comment:
I don’t think all of these tests should be removed, one set of them should be refactored to not provide `peekGuid` or `GUID_KEY` but still confirm the expectation that the method is only called once.

Reply:
returned 2 tests back others are just duplicates